Calibration/TkAlCaRecoProducers/python/ALCARECOSiStripPCLHistos_cff.py

```diff
@@ -9,7 +9,6 @@
                                        # 2 provide more detailed output
                                        Frequency = cms.untracked.int32(50),
                                        MEPathToSave = cms.untracked.string('AlCaReco/SiStrip'),
-                                       deleteAfterCopy = cms.untracked.bool(False)
```

Comment:
running the step2 of wf 1040 with the ```ALCAPRODUCER:@allForExpress``` option:
![express](https://user-images.githubusercontent.com/4058194/69442998-d2e3aa80-0d4d-11ea-9654-9df955b2be69.png)

Reply:
@fabiocos, I am sorry to be picky, but I don't think `@allForExpress` resolves into the `PromptCalibProd...` ALCAPROMPT producer changed here, so I am afraid your plot proves nothing. 
A more interesting test IMHO is to monitor the RSS of step3 of wf 1001.0:
```
cmsDriver.py step3  --datatier ALCARECO --triggerResultsProcess RECO --conditions
auto:run1_data -s ALCAOUTPUT:SiStripCalZeroBias+TkAlMinBias+DtCalib+Hotline+LumiPix
elsMinBias+AlCaPCCZeroBiasFromRECO+AlCaPCCRandomFromRECO,ALCA:PromptCalibProd+Promp
tCalibProdSiStrip+PromptCalibProdSiStripGains+PromptCalibProdSiStripGainsAAG+Prompt
CalibProdSiPixelAli+PromptCalibProdSiPixel --eventcontent ALCARECO -n 100  --filein
  file:step2.root  --fileout file:step3.root  
```

Reply:
@mmusich as far as I can see in also the Tracer dump it do runs both ```MEtoEDMConvertSiStrip``` and ```pathALCARECOSiStripPCLHistos``` that are also modified in this PR

Reply:
What about `PromptCalipProdSiStripGais...` and `PromptCalibProdSiPixelAli`?

Reply:
Or your test was limited to this particular file change?

Reply:
@mmusich I believe that we are clearly missing a test that is realistically mimicking what happens nowadays at the T0 , not bits and pieces. My understanding from a discussion with @tocheng is that ```@allForExpress``` should cover all the ALCAPRODUCER available, and I looked at a few ALCAPROMPT touched by this PR separately. 
It would be important to get from ALCA  a realistic workflow on recent data. Anyway I will make a final check following your suggestion (but is this test on very old data representative of what happens in today's situation?)

Reply:
Thanks for the additional tests. I agree about the lack of a suitable relval test testing all the AlCa aspects of Express.
For tracker at least 1001.0 is a reasonable proxy. For the changes proposed here I don't think occupancy or n. of channel differences matter to measure a comparison of the memory consumption (it would be nice to have 1001.0 updated with Run2 input data though).

Reply:
running wf 1001 on 5000 events, the comparison of RSS used (by a 4-threads jobs) by step3 is

![express2011](https://user-images.githubusercontent.com/4058194/69497904-271f9380-0ee2-11ea-8b61-05d240860be9.png)